src/tabs/NPSP_Resources.tab

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <CustomTab xmlns="http://soap.sforce.com/2006/04/metadata">
-    <label>Getting Started</label>
+    <label>Get Started</label>
```

Comment:
This is the current (old) Getting Started page, and should remain as "Getting Started".